_layouts/landing.html

```diff
@@ -118,25 +118,42 @@ <h5 class="title">Reference</h5>
         <div class="cardlet container-fluid" style="min-height:307px">
           <div class="row">
             <div class="col-xs-12 col-lg-8">
-              <h5 class="title">How do I?</h5>
-              <p>
-                Check out the following topics to learn how to build, run, and
-                deploy your applications using Docker.
-              </p>
+              <h5 class="title"></h5>
             </div>
           </div>
           <div class="row">
-            <div class="col-xs-12 col-md-6"><a href="/language/">Containerize language-specific apps using Docker</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/engine/reference/builder/">Write a Dockerfile</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/network/">Manage container networking</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/compose/compose-file/">Write a Docker Compose file</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/storage/">Work with volumes and bind mounts</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/get-started/04_sharing_app/">Share my image on Docker Hub</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/config/daemon/">Configure the Docker daemon</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/get-started/02_our_app/">Build and run an image</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/config/labels-custom-metadata/">Manage Docker objects</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/build/building/multi-stage/">Multi-stage builds</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/get-started/swarm-deploy/">Scale apps using Swarm</a></div>
+            <ul class="nav nav-tabs">
+              <li class="active"><a data-toggle="tab" data-target="#includes">Explore Docker</a></li>
```

Comment:
Looks like there's an indentation issue in the Explore section. Could you pls fix this?

![image](https://user-images.githubusercontent.com/47779042/198986430-b805c4f5-b679-453a-b0d0-3cee32f1649d.png)